modules/mobile/app/services/mobile/v0/lighthouse_letters/service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module LighthouseLetters
+      class Service < Common::Client::Base
+        configuration Configuration
+
+        # rubocop:disable Lint/MissingSuper
```

Comment:
Is there a reason for not calling super?

Reply:
it seems that it's a not often desired rubocop rule. there's a large list of files that exclude this rule in the `.rubocop_todo.yml` file. but it took me a while to figure out that's why it was giving me a linting error while other examples of initialize methods in our code don't call super. I'm use to the rules being disabled/enable inline so figured this was a better for future reference to other parts of the code. I'm good with just including it in the excludes rule file though. I also just tried including super but was only getting errors.

Reply:
Yeah, it looks like someone tried to clean up all the inline linting about a year ago by moving it into that excludes file. Most likely, the developers just forgot to call super, then when we added/updated the linter someone just put a bunch of exceptions into this exclude file rather than fix the problems. It's also possible some of these are examples of "inheritance gone wrong" because a subclass should always be able to call super safely. 

The issue is that the subclass's `initialize` has an argument `user` while the parent does not. When you call super, it automatically includes the arguments. You can fix this by calling `super()` with parens, which calls the method without arguments. There's a good chance this is why so many people have ignored this linting rule.

Reply:
ahh, I forgot that `super` passed the same arguments by default without the (). Thanks for looking into this. fixed.

Reply:
Based on how many times we've ignored this linting rule, I don't think you're alone in that.